pep-0685.rst

```diff
@@ -12,75 +12,81 @@ Post-History: 08-Mar-2022
 Abstract
 ========
 
-This PEP specifies how to normalize `distribution _extra_ <Provides-Extra_>`_
+This PEP specifies how to normalize `distribution extra <Provides-Extra_>`_
 names when performing comparisons.
-This prevents tools from either failing to find an extra name or
+This prevents tools from either failing to find an extra name, or
 accidentally matching against an unexpected name.
 
 
 Motivation
 ==========
 
-The `Provides-Extra`_ core metadata specification says that an extra's
+The `Provides-Extra`_ core metadata specification states that an extra's
 name "must be a valid Python identifier".
-:pep:`508` says that the value of an ``extra`` marker may contain a
+:pep:`508` specifies that the value of an ``extra`` marker may contain a
 letter, digit, or any one of ``.``, ``-``, or ``_`` after the initial character.
-Otherwise there is no other specification at https://packaging.python.org
+Otherwise, there is no other `PyPA specification
+<https://packaging.python.org/en/latest/specifications/>`_
 which outlines how extra names should be written or normalization for comparison.
-Due to the amount of packaging-related code out there,
+Due to the amount of packaging-related code in existence,
 it is important to evaluate current practices by the community and
-standardize on a practice that doesn't break most code while being
+standardize on one that doesn't break most code, while being
 something tool authors can agree to following.
 
-The issue of no standard was brought forward via the discussion at
-https://discuss.python.org/t/what-extras-names-are-treated-as-equal-and-why/7614
-where the extra ``adhoc-ssl`` was not considered equal to the name
+The issue of there being no standard was brought forward by an
+`initial discussion <https://discuss.python.org/t/7614>`__
+noting that the extra ``adhoc-ssl`` was not considered equal to the name
 ``adhoc_ssl`` by pip.
 
 
 Rationale
 =========
 
-:pep:`503` specifies how to normalize distribution names:
-``re.sub(r"[-_.]+", "-", name).lower()``.
+:pep:`503` specifies how to normalize distribution names::
+
+    re.sub(r"[-_.]+", "-", name).lower()
+
 This collapses any run of the substitution character down to a single
 character,
 e.g. ``---`` gets collapsed down to ``-``.
 This does not produce a valid Python identifier as specified by the
 core metadata specification for extra names.
 
 `Setuptools does normalization <https://github.com/pypa/setuptools/blob/b2f7b8f92725c63b164d5776f85e67cc560def4e/pkg_resources/__init__.py#L1324-L1330>`__
-via ``re.sub('[^A-Za-z0-9.-]+', '_', name).lower()``.
+via::
+
+    re.sub('[^A-Za-z0-9.-]+', '_', name).lower()
+
 The use of an underscore/``_`` differs from PEP 503's use of a
 hyphen/``-``.
 Runs of characters, unlike PEP 503, do **not** get collapsed,
 e.g. ``___`` stays the same.
 
 For pip, its
-"extra normalisaton behaviour is quite convoluted and eratic",
+"extra normalisation behaviour is quite convoluted and erratic",
```

Comment:
This appears to be a verbatim quote (given the BrE and multiple typos), so it would be beneficial to link or at least state the source for said quote, to lend it appropriate credibility.